app/src/main/java/org/mozilla/focus/settings/PrivacySecuritySettingsFragment.kt

```diff
@@ -35,10 +45,23 @@ class PrivacySecuritySettingsFragment : BaseSettingsFragment(),
 
     override fun onSharedPreferenceChanged(sharedPreferences: SharedPreferences, key: String) {
         TelemetryWrapper.settingsEvent(key, sharedPreferences.all[key].toString())
+        updateStealthToggleAvailability()
     }
 
-    companion object {
+    private fun updateStealthToggleAvailability() {
+        val switch =  preferenceScreen.findPreference(resources.getString(R.string.pref_key_secure)) as SwitchPreference
+        if (preferenceManager.sharedPreferences.getBoolean(resources.getString(R.string.pref_key_biometric), false)) {
+            preferenceManager.sharedPreferences.edit().putBoolean(resources.getString(R.string.pref_key_secure), true).apply()
```

Comment:
\[detekt\] INFO: \[style/MaxLineLength\] Line detected that is longer than the defined maximum line length in the code style\.